cmd/kyverno/main.go

```diff
@@ -92,13 +92,22 @@ var (
 	admissionReports           bool
 	reportsChunkSize           int
 	logFormat                  string
+	//
+	dumpPayload      bool
+	dumpPayloadTimer string
```

Comment:
What is this timer ?

Reply:
it was mentioned on the issue page, i wanted to ask about it
https://github.com/kyverno/kyverno/issues/1334#issuecomment-850040073
https://github.com/kyverno/kyverno/issues/1334#issuecomment-850362773

Reply:
yeah, don't bother with the timer